test/electionguard/benchmark/bench_encrypt.cpp

```diff
@@ -83,12 +84,13 @@ class EncryptBallotFixture : public benchmark::Fixture
   public:
     void SetUp(const ::benchmark::State &state)
     {
-        keypair = ElGamalKeyPair::fromSecret(TWO_MOD_Q());
-        manifest = ManifestGenerator::getJeffersonCountryManifest_multipleBallotStyle_fromFile();
+        nonces = make_unique<Nonces>(TWO_MOD_Q(), "encrypt-ballot-benchmark");
+        keypair = ElGamalKeyPair::fromSecret(*nonces->get(0));
+        manifest = ManifestGenerator::getManifestFromFile(TEST_SPEC_VERSION, TEST_USE_SAMPLE);
```

Comment:
Just pointing out that in other places where the defaults were used, there were no parameters used.  The defaults are what are used here.  Not sure this needs to change here since the defaults could change in the future.